.github/CODEOWNERS

```diff
@@ -3,39 +3,28 @@
 # For details on acceptable file patterns, please refer to the [Github Documentation](https://help.github.com/articles/about-codeowners/)
 
 # default owners, overridden by package specific owners below
-* @asaj @cmcewen @timmoreton
+* @celo-org/default-owners
 
 # directory and file-level owners. Feel free to add to this!
-/packages/azure/ @codyborn
-/packages/blockchain-api/ @jmrossy @cmcewen @i1skn @jeanregisser
-/packages/celotool/ @asaj
-/packages/cli/ @mcortesi @gastonponti @medhak1 @celo-org/devx
-/packages/sdk/ @mcortesi @gastonponti @medhak1 @barbaraliau @celo-org/devx
-/packages/dappkit/ @cmcewen @barbaraliau @celo-org/devx
-/packages/dev-utils/ @mcortesi
-/packages/docs/ @timmoreton @barbaraliau @critesjosh 
-/packages/faucet/ @eelanagaraj @celo-org/devx
-/packages/mobile/ @celo-org/wallet 
-/packages/moonpay-auth/ @annakaz @i1skn 
-/packages/notification-service/ @annakaz @cmcewen @i1skn @jeanregisser
-/packages/flake-tracker @alecps
 
-/packages/protocol/ @celo-org/cap
-/packages/sdk/contractkit/ @celo-org/cap
-/packages/komencikit/ @celo-org/cap
-/packages/phone-number-privacy/ @celo-org/cap
-/packages/attestation-proxy/ @celo-org/cap
-/packages/attestation-service/ @celo-org/cap
-/packages/metadata-crawler/ @celo-org/cap
+/packages/attestation-proxy/ @celo-org/identity
+/packages/attestation-service/ @celo-org/identity
+/packages/azure/ @celo-org/devopsre
+/packages/celotool/ @celo-org/protocol-owners
+/packages/cli/ @celo-org/applications
+/packages/dev-utils/ @celo-org/applications
+/packages/docs/ @celo-org/applications @celo-org/devrel
 /packages/env-tests @celo-org/cap
-
-/packages/react-components/ @i1skn @jeanregisser @celo-org/wallet
-/packages/terraform-modules-public/ @tkporter
-/packages/terraform-modules/ @tkporter
-/packages/typescript/ @cmcewen
-/packages/sdk/utils/ @barbaraliau @medhak1 @celo-org/devx
-/packages/sdk/base/ @barbaraliau @medhak1 @celo-org/devx
-
-# helm-charts rules.
-/packages/helm-charts/ @nambrot @timmoreton
-/packages/helm-charts/oracle/ @celo-org/platform-economics
+/packages/faucet/ @celo-org/applications
+/packages/flake-tracker @celo-org/flake-trackers
+/packages/helm-charts/ @celo-org/devopsre
+/packages/helm-charts/oracle/ @celo-org/platform-economics  @celo-org/devopsre
```

Comment:
This might need a new team (issuances & oracles) + I would rather make this one explicitly owned by the circle managing the oracles (removing devopsre)

Reply:
Oh well, my bad for not paying attention to the automerge label...
In any case, this is an improvement over the previous situation!